netsim/__init__.py

```diff
@@ -6,14 +6,18 @@
 
 abort = False
 
+if sys.version_info < (3,9):
+  print(f"FATAL ERROR: Netlab requires Python version 3.9 or newer - installed {sys.version}")
+  abort = True
+
 try:
   import box
 
-  if box.__version__ < '7.2.0':                 # netlab needs Python Box version 7.0 or higher
+  if box.__version__ < '7.2.0':                 # netlab needs Python Box version 7.2 or higher
```

Comment:
And you chose to change the minimum Box version as well. I wonder what the justification was.

Actually, I don't want to know.